### PR TITLE
Prefill name, email, and organization on Feedback form

### PR DESCRIPTION
**Pull request by exu3**
Name, Email, and Organization fields are prefilled if the user clicks on the Feedback form link in the navbar.

### CODE REVIEW

File: app/views/users/_nav.html.erb [hunk 1]
```diff
@@ -16,7 +16,7 @@
       'payment-docs', my_inbox_path %>
     <%= dock_item 'Feedback',
       'Submit user suggestions.',
-      'message-new', "https://hack.af/bank-userfeedback" %>
+      'message-new', "https://hack.af/bank-userfeedback?prefill_Name=#{current_user.name}&prefill_Email=#{current_user.email}&prefill_Organization=#{current_user.events.first.name}" %>
```
Comment (cjdenio) [inline app/views/users/_nav.html.erb]: You may want to handle the very-edge-case situation where the current user isn't in any organizations. (`current_user.events.first&.name`)

###### :no-organizations-head-empty:

Comment (exu3) [reply app/views/users/_nav.html.erb]: Ah good call, thanks!

**Comment by hackliv** [comment]
Hey how does this work if the user is a part of numerous orgs?

**Comment by exu3** [comment]
@hackliv It will prefill with the event whose ID is first numerically. For example, Hack Club HQ (183) comes before Hack Club Bank (636).

**Comment by hackliv** [comment]
@exu3 perfect! thanks for clarification

**Review by hackliv — APPROVED** [review]

**Pull request merged**